### DISCUSSION TITLE
tutorial-extras/adc

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#10] [comment]
# tutorial-extras/adc

简介 本章介绍使用arduino烧录Air001开发板测量ADC。 硬件准备 1. 按☁️ Air001开发板入门 (/tutorial-advanced/Air001_start.html)，将Air001和DAPLink调试器使用排针排母连接。 2. 可控电压源，负极接Air001的GND引脚(地)、正极接引脚PA_0(ADC)。 软件部分 按前文...

https://arduino.luatos.com/tutorial-extras/adc.html

**Comment by yangliangbin** [comment]
这个示例我这边ADC不行，串口有打印，，请问啥问题？

**Reply by chenxuuu** [reply]
测试无问题，如有BUG请前往ISSUE上报：https://github.com/Air-duino/Arduino-AirMCU/issues

**Reply by yangliangbin** [reply]
我这边用的电位器，接的信号，GND和3.3V，，信号接的PA0，，，串口一直打印0，，怎么旋转电位器都没有反应

**Reply by chenxuuu** [reply]
你可以量一下实际电压，如有问题，请提issue，文档评论区仅用于讨论文档内容

**Comment by Mr-yang920** [comment]
Air001的ADC为12bit,望改正

**Reply by chenxuuu** [reply]
要看实际初始化的配置是多少了

**Reply by HalfSweet** [reply]
可以参考<https://arduino.luatos.com/library/adc.html#analogreadresolution>进行设置，为了兼容性默认设置是10bit

**Comment by Hello-2877515114** [comment]
我这个为什么下载程序ADC是准的，掉电之后 ADC就不准了呢
